website/content/en/v0.28/troubleshooting.md

```diff
@@ -314,6 +314,35 @@ When a node is launched by Karpenter, it is assigned to a subnet within your VPC
 
 For more troubleshooting information on why your pod may have a `FailedCreateSandbox` error, view the [EKS CreatePodSandbox Knowledge Center Post](https://repost.aws/knowledge-center/eks-failed-create-pod-sandbox).
 
+#### Windows pods are failing to launch
+
+The following solution(s) may resolve your issue if you are seeing an error similar to the following when attempting to launch Windows pods.
+
+```
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx": plugin type="vpc-bridge" name="vpc" failed (add): failed to parse Kubernetes args: pod does not have label vpc.amazonaws.com/PrivateIPv4Address
+```
+
+This error typically occurs if you have not enabled Windows support. If you are provisioning Windows nodes, you need to [enable Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for your cluster by enabling Windows IPAM.
+
+##### Solutions
+
+1. Set `enable-windows-ipam` to true in the `amazon-vpc-cni` config map. See documentation [Enabling Windows support](https://docs.aws.amazon.com/eks/latest/userguide/windows-support.html#enable-windows-support) for more details regarding how this can be accomplished.
+
+### Windows pods fail to launch with image pull error
+
+The following solution(s) may resolve your issue if you are seeing an error similar to the following when attempting to launch Windows pods.
+
+```
+Failed to pull image "mcr.microsoft.com/windows/servercore:xxx": rpc error: code = NotFound desc = failed to pull and unpack image "mcr.microsoft.com/windows/servercore:xxx": no match for platform in manifest: not found
+```
+
+This error typically describes a scenario whereby a pod with a given container OS version attempts to be scheduled on an incompatible Windows host OS version.
+Windows requires the host OS version to match the container OS version.
+
+##### Solutions
+
+1. Define your `Provisioner` requirements and pod `nodeSelector` such that your containers are scheduled on a compatible OS host version. You can read more on Windows container version compatibility in [container docs](https://learn.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility). See Windows `Provisioner` [examples here](https://github.com/aws/karpenter/blob/main/examples/provisioner/windows.yaml).
```

Comment:
Provisioner requirements will be added automatically based on the amiFamily selected in NodeTemplate. The mitigation here is to use the correct ami family in the node template and use `nodeSelector` in the workload manifests to direct the pod to specific node.

Reply:
Ack